### ISSUE TITLE
DLVFX supported resolutions?

### ISSUE THREAD

**Issue by Selur** [AkarinVS/vapoursynth-plugin#27]
using:`clip = core.akarin.DLVFX(clip, op=0, strength=1, model_dir='f:/Support/Akarin/models')`
works fine for 1920x1080 content, using 4k content, I get:
```
failed VFX call NvVFX_Load(d->vfx): fffffff6 (Error while loading the TRT model)
```
using for example 960x540, I get:
```
failed VFX call NvVFX_SetImage(d->vfx, NVVFX_INPUT_IMAGE, &d->srcGpuImg): ffffff90 (invalid pitch argument)
```
=> Does DLVFX only work with 1920x1080 content?

**Comment by WolframRhodium** [comment]
> `op=0`: ... input requirements: `90 <= height <= 1080`, **`256 <= width <= 1920+128`**, **`width % 128 == 0`**.

**Comment by Selur** [comment]
Thanks!

**Comment by couleurm** [comment]
> > `op=0`: ... input requirements: `90 <= height <= 1080`, **`256 <= width <= 1920+128`**, **`width % 128 == 0`**.

can you explain in laymans term?

**Comment by Selur** [comment]
Height need to be at least 90 and at most 1080 pixels.
Width needs to be at least 256 and at most 1920+128=2048 pixels, also width needs to be a multiple of 128.
